src/main/java/sirius/kernel/settings/Settings.java

```diff
@@ -267,18 +269,77 @@ public Duration getDuration(String path) {
     /**
      * Returns the string for the given key.
      *
-     * @param key the key used to lookup the string value
+     * @param key the key used to look up the string value
      * @return the string value stored of the given key
      */
     @Nonnull
     public String getString(String key) {
         return get(key).asString();
     }
 
+    /**
+     * Returns a translated string for the given key.
+     * <p>
+     * Based on the available config section at <tt>key</tt> this will perform the appropriate translation task:
+     *     <ul>
+     *         <li>
+     *             If the key doesn't exist, <tt>""</tt> will be returned. Note that if this config is <tt>strict</tt>,
+     *             an error is logged if the requested path does not exist. However, no exception will be thrown.
+     *         </li>
+     *         <li>If the key points to a plain string literal, this will be returned</li>
+     *         <li>
+     *             If the key points to a string literal starting with "$", an {@link sirius.kernel.nls.NLS#get(String)}
+     *             look up for the value will be performed.
+     *         </li>
+     *         <li>
+     *             If the key points to a config map, the value with the current language as key or the <tt>default</tt>
+     *             value will be used.
+     *         </li>
+     *     </ul>
+     * </p>
+     *
+     * @param key  the key used to look up the string value
+     * @param lang the language to fetch the translation for. Use <tt>null</tt> to indicate that the
+     *             {@link NLS#getCurrentLang() current language} should be used.
+     * @return the translated string
+     */
+    @SuppressWarnings("unchecked")
+    @Nonnull
+    public String getTranslatedString(String key, @Nullable String lang) {
+        Value value = get(key);
+        if (value.isFilled() && value.is(String.class)) {
+            return NLS.smartGet(value.asString(), lang);
+        }
+
+        if (value.is(Map.class)) {
+            Map<String, String> translations = value.get(Map.class, Collections.emptyMap());
+            String effectiveLanguage = Strings.isEmpty(lang) ? NLS.getCurrentLang() : lang;
+            return Optional.ofNullable(translations.get(effectiveLanguage))
+                           .or(() -> Optional.ofNullable(translations.get("default")))
```

Comment:
Should we name this "fallback" so we don't have to do any mappings/transformations when storing a MultiLanguageString to a config (via UI)?

Reply:
Do we have any `Config` -> `String` transformation anywhere?
Naming it "default" makes perfekt sense to me, as the fallbacks in `Extension`s are also called "default"

Reply:
For extensions we did us "default" so it felt more natural than using fallback. However, as these things are unrelated, we can also use fallback.